frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1047,7 +1049,7 @@ export default ({
             $state.complete()
             if (!this.$refs.conversation ||
             this.$refs.conversation.scrollHeight === this.$refs.conversation.clientHeight) {
-              const msg = this.messages[this.messages.length - 1]
+              const msg = this.messages.filter(m => !m.pending).slice(-1)
```

Comment:
Also here, slice(-1) returns array. And why is it needed to filter by `pending` here?

Reply:
> Also here, slice(-1) returns array.
Correct, that should be fixed.
> Why do you think the last message could be pending
I'd need to double check, as for the other similar question. But the fact was that some messages that were pending were being added to the last unread, which was then causing issues loading the chatroom. Hence, at least at one location the messages were pending.